src/components/ui/Breadcrumb/breadcrumb.scss

```
@@ -41,21 +66,94 @@
 
     [data-breadcrumb-divider] {
       height: var(--fs-spacing-3);
-      margin: 0 var(--fs-spacing-2) 0 var(--fs-spacing-1);
+      margin: 0 var(--fs-spacing-1) 0 var(--fs-spacing-1);
       border-left: var(--fs-border-width-default) solid var(--fs-border-color-light);
     }
 
     [data-store-link] {
-      display: inline;
       padding: 0;
 
       &:visited {
         color: var(--fs-color-link);
       }
     }
 
+    [data-store-dropdown-button] {
+      display: inline;
+      margin: calc(var(--fs-spacing-0) / 4);
+      padding: 0;
+      padding: 0 calc(var(--fs-spacing-0) / 2);
```

There are 2 of them.
```suggestion
      padding: 0 calc(var(--fs-spacing-0) / 2);
```